spec/models/recipe_spec.rb

```diff
@@ -2,7 +2,7 @@
 
 RSpec.describe Recipe, type: :model do
```

Comment:
I'd love to see a spec that tests the scope too..... let me know if you want to chat about this.